app/assets/stylesheets/modules/mfa.css

```diff
@@ -7,6 +7,29 @@
   flex: 1;
 }
 
+.mfa__header-wrapper {
+  display: flex;
+  align-items: center;
+  gap: 12px;
+}
+
+.t-body .mfa__header {
+  display: inline-block;
+  padding-top: 0px;
+}
+
+.badge {
+  border-radius: 5px;
+  padding: 5px 10px;
+  text-transform: uppercase;
+  font-weight: normal;
+  font-size: 12px;
+  display: block;
+  background-color: #fff6d2;
+  border: 1px solid #e9d893;
```

Comment:
Curious on why a yellow was chosen, to me it feels like a "warning". I was playing around with the colours and I prefer as it is more inline with the RubyGems colour scheme
```
background-color: #ffe6d4;
border: 1px solid #e9573f;
```
<img width="842" alt="Screenshot 2023-07-17 at 10 22 39 AM" src="https://github.com/rubygems/rubygems.org/assets/42748004/1c93333e-9545-4567-9711-f5aa7bf7c816">

Reply:
Good point! I think this color is a lot more suitable